docs/CHANGELOG.rst

```diff
@@ -44,17 +50,10 @@ Fixed
   process is finished
 - Remove unrelated permissions occasionally listed among group
   permissions on ``permissions`` endpoint
-- Fix ``ResolwePermissionsMixin`` to work correctly with multi-words 
-  model names, i.e. DescriptorSchema
-
-
-==================
-3.0.1 - 2017-09-15
-==================
-
-Fixed
------
-- Correctly relabel SELinux contexts on user/group files
+- Fix ``ResolwePermissionsMixin`` to work correctly with multi-words
+  model names, i.e. ``DescriptorSchema``
+- Fix Elasticsearch indices to work when running tests in parallel
```

Comment:
This is actually not true, as old version worked with Django 1.10, which was required by Resolwe. It was changed to keep the functionality when requirement was bumped to Django 1.11. But from user"s point of view nothing has changed.

Reply:
Ok, I see. I've removed the CHANGELOG.

I would ask you to put this info to the corresponding commit message next time, as it helps understand why the change was made.